test/async-hooks/test-promise.js

```diff
@@ -46,7 +46,8 @@ function onexit() {
   const a1 = as[1];
   assert.strictEqual(a1.type, 'PROMISE', 'promise request');
   assert.strictEqual(typeof a1.uid, 'number', 'uid is a number');
-  assert.strictEqual(a1.triggerId, 1, 'parent uid 1');
+  assert.strictEqual(a1.triggerId, a0.uid,
+                     'child promise should use parent uid as triggerId');
```

Comment:
Yeah, I would just not use an error message here (i.e. `assert.strictEqual(a1.triggerId, a0.uid);`). Sorry the tests are written this way.